pkg/template/resource.go

```diff
@@ -111,7 +111,7 @@ func ApplyParamsToResourceTemplate(params []triggersv1.Param, rt json.RawMessage
 // param value substituted for all matching param variables in the template
 func applyParamToResourceTemplate(param triggersv1.Param, rt json.RawMessage) json.RawMessage {
 	// Assume the param is valid
-	paramVariable := fmt.Sprintf("$(params.%s)", param.Name)
+	paramVariable := fmt.Sprintf("$(tt.params.%s)", param.Name)
```

Comment:
Are we just supporting `$(tt.params)` for now.  Shouldn't we support both `$(params)` or `$(tt.params)` for a release or two?

Reply:
Updated changes
Please do have a look 
Thank you